File: GdPicture/WinForm/C#/DICOM Viewer/frmProperties.cs
using System;
using System.Windows.Forms;
using GdPicture14;

namespace GdPicture_DICOM_Viewer
{
    public partial class frmProperties : Form
    {
        public frmProperties()
        {
            InitializeComponent();
        }

        private void frmProperties_Load(object sender, EventArgs e)
        {
            int imageId = ((frmMain)Form.ActiveForm).NativeImageId;
            using (GdPictureImaging oGdPictureImaging = new GdPictureImaging())
            {
                int tagsCount = oGdPictureImaging.DicomGetTagsCount(imageId);
                for (int i = 0; i <= tagsCount - 1; i++)
                {
                    string tagID = "<" + String.Format("{0:X}", (oGdPictureImaging.DicomGetTagGroup(imageId, i))).PadLeft(4, '0') + "," + String.Format("{0:X}", (oGdPictureImaging.DicomGetTagElement(imageId, i))).PadLeft(4, '0') + ">";
                    DataGridView1.Rows.Add(new object[] { tagID, oGdPictureImaging.DicomGetTagDescription(imageId, i), oGdPictureImaging.DicomGetTagValue(imageId, i) });
                }
            }
        }
    }
}

File: GdPicture/WinForm/vb.net/DICOM Viewer/frmProperties.vb
Imports GdPicture14

Public Class frmProperties

    Private Sub frmProperties_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Dim ImageID As Integer = frmMain.getNativeImage()
        Using oGdPictureImaging As New GdPictureImaging
            Dim tagsCount As Integer = oGdPictureImaging.DicomGetTagsCount(ImageID)
            For i As Integer = 0 To tagsCount - 1
                Dim tagID As String = "<" + Hex(oGdPictureImaging.DicomGetTagGroup(ImageID, i)).PadLeft(4, "0"c) + "," + Hex(oGdPictureImaging.DicomGetTagElement(ImageID, i)).PadLeft(4, "0"c) + ">"
                DataGridView1.Rows.Add(New Object() {tagID, oGdPictureImaging.DicomGetTagDescription(ImageID, i), oGdPictureImaging.DicomGetTagValue(ImageID, i)})
            Next
        End Using
    End Sub

End Class